src/commands/shell.rs

```diff
@@ -23,11 +26,15 @@ pub async fn shell(_: Shell, global: GlobalArgs) -> crate::error::Result<()> {
         &tempfile,
         format!("source {}", env.activate_path().to_string_lossy()),
     )?;
-    tokio::process::Command::new("bash")
-        .arg("--rcfile")
-        .arg(tempfile.path())
-        .spawn()?
-        .wait()
-        .await?;
+
+    let mut command = tokio::process::Command::new("bash");
+    command.current_dir(&global.project);
+    command.arg("--rcfile").arg(tempfile.path());
+    if !args.command.is_empty() {
+        command.arg("-c").arg(args.command.join(" "));
```

Comment:
I don't know if its that simple to just join with spaces because the quotations would be missing or?

Reply:
might just allow the user to either pass a string to `-c` or treat everything in the slop as args to bash

Reply:
yeah it just won't work though. like if i have a file named `file with spaces.txt` and I do `aqora shell -- cat "file with spaces.txt"` it wouldn't work. and also I can't pass things like pipe operators or other stuff. I mean I agree the `-c` is annoying but there's good reason

Reply:
Ah yes totally! I simplified this command a bit